Library/Formula/vc4asm.rb

```diff
@@ -17,7 +17,12 @@ class Vc4asm < Formula
 
   patch do
     url "https://github.com/maazl/vc4asm/pull/2.patch"
-    sha256 "9b7996563a6b15ae7d5578df6d08b43413dc1758e8e2002115ba414daed323e3"
+    sha256 "589fd37c768861a414ed8968a5af699154283cc8be63fae7b12db422db944cb4"
+  end
```

Comment:
I'd rather we remove patches that aren't causing actual failures (similar with the one below that looks like a feature rather than a bug)